library/core/src/macros/mod.rs

```diff
@@ -1577,8 +1577,12 @@ pub(crate) mod builtin {
 
     /// Unstable implementation detail of the `rustc` compiler, do not use.
     #[rustc_builtin_macro]
-    #[stable(feature = "rust1", since = "1.0.0")]
-    #[allow_internal_unstable(core_intrinsics, libstd_sys_internals, rt)]
+    #[unstable(
+        feature = "rustc_encodable_decodable",
+        issue = "none",
+        soft,
+        reason = "unstable implementation detail of the `rustc` compiler, do not use"
```

Comment:
This is incorrect, it's not an implementation detail of the compiler. Can you put a more accurate message here?

Reply:
There is a comment immediately above this code that has identical wording. Is that comment incorrect?

Reply:
Yeah, rustc hasn't used these macros for a while now, it uses its own derive proc macros and its own rustc-serialize fork in-tree.

Reply:
Alright, that's been updated. Should be good to merge now! Looking into the edition lint, but there's tons of time for that.